### ISSUE TITLE
Update Documentheader - 2 badges en extra knop

### ISSUE THREAD

**Issue by MichielDHVD** [dso-toolkit/dso-toolkit#1515]

**Comment by DjangoBlok** [comment]
Oei hier gaat iets fout in communicatie. Ik had in overleg de story in Jira ge-update met een nieuw ontwerp. Maar die update wordt natuurlijk niet automatisch doorgezet naar github.
@daverhelst moet ik Github nu updaten met de laatste stand van zaken? (dat is dus een iets andere stand dan wat je hebt gebouwd)

**Comment by daverhelst** [comment]
Dank @DjangoBlok, ik heb de vulling aangepast naar de nieuwe ontwerpen. Maar mijn eerdere vraag blijft wel staan. De vulling is nu opgebouwd uit bestaande toolkit-elementen.

In hoeverre moeten deze als onderdeel van de document header bestaan?

**Comment by DjangoBlok** [comment]
Bedoel je met vulling, de informatie die in de opgengeklapte panels is te zien?
Ja lijkt me goed om dat zo veel mogelijk met bestaande toolkit componenten op te bouwen. 
Ik zie wel dat je wat classes hebt toegevoegd? (zoals documentheader-status-item) 
Een paar optische bevindingen:

- Die lijstjes met versies voelen wel wat te krap. 
- Zou je in plaats van de badges de label compact kunnen gebruiken? (behalve bij de twee badges met cijfertjes in de header)
- Die knop "overige versies" graag op dezelfde manier (zonder kader) als de knop "meer kenmerken"
- Tussen "een omgevingsplan waar.... mooier van wordt" en "meer kenmerken" zit nu meer ruimte dan tussen "meer kenmerken" en de regel eronder. Is dat gelijk te trekken?

**Comment by gofix** [comment]
De nu falende test faalt niet voor niets, die probeert linkjes te focusen die niet meer in de voorbeeldcontent voorkomen.

**Comment by daverhelst** [comment]
@DjangoBlok @LiesbethVorst Het heeft even geduurd, maar er is wat te zien:

https://core.dso-toolkit.nl/_1515.Update-Documentheader/?path=/story/viewer-grid--viewer-grid

We hebben de verschillende "smaken" in de juiste stories onder gebracht. Zijn jullie het eens met deze onderverdeling en hoe het er uit ziet/en of werkt?

**Comment by DjangoBlok** [comment]
Heel mooi @daverhelst Dit is inderdaad wat ik voor ogen had.
Zou je op voorbeeldpagina-document-item wel iets van ruimte kunnen creëren tussen de twee losse instanties? (geen margin toevoegen, maar alleen op de voorbeeldpagina)

**Comment by daverhelst** [comment]
@DjangoBlok dat gaat helaas niet lukken zonder toevoeging van stijling op `.dso-document-list`, het is namelijk een `<ul>`, ik zag dat het in de oude situatie ook niet aanwezig was.

**Comment by DjangoBlok** [comment]
Dat was ook in de oude situatie niet ideaal. 
Als het een UL is, dan zou het wel handig zijn om ook aan te geven hoe onderliggende content erin moet komen. Eventueel met een zo'n blauwe infobox zoals op het grid zelf?

**Comment by LiesbethVorst** [comment]
@daverhelst wat me nog opvalt bij de badges in de header, is dat de versie met een outline 24px hoog is. Dat lijkt me correct, maar de andere badges zijn maar 22px hoog.

**Comment by daverhelst** [comment]
@LiesbethVorst dat klopt wel. De nieuwe versie heeft een border gekregen, deze is dus eigenlijk 2px te groot. Daar zal ik de maat voor moeten aanpassen.

**Comment by daverhelst** [comment]
Zoals besproken met Django:

De situatie op de `document item` voorbeeldpagina is nog incompleet. Per item zullen er nog regels bij moeten komen. Deze wens wordt in een nieuw issue beschreven.

**Comment by gofix** [comment]
@DjangoBlok @LiesbethVorst vergelijk
Viewer Grids stories oud:
https://core.dso-toolkit.nl/38.1.1/?path=/story/viewer-grid--with-search-results
![image](https://user-images.githubusercontent.com/2689559/174823579-a2ccb7b9-a442-4f07-9ede-7a5cbd42bd03.png)
Viewer Grids stories nieuw:
https://core.dso-toolkit.nl/_1515.Update-Documentheader/?path=/story/viewer-grid--viewer-grid
![image](https://user-images.githubusercontent.com/2689559/174823674-9dece65d-f861-43b3-abe1-5f7665ed3dbc.png)

**Comment by DjangoBlok** [comment]
Ja, ziet er goed uit. Ik heb er nog een keer doorheen gelopen en zie niks blokkerends.

**Comment by LiesbethVorst** [comment]
We hebben toch nog wat details gespot:
![git 1515-2](https://user-images.githubusercontent.com/91903058/174839407-6b172390-fe6f-4a83-b114-6a3b318e5a6f.png)
![git 1515-1](https://user-images.githubusercontent.com/91903058/174839425-0a321658-e011-4dcb-acc9-a9a0a8ec605c.png)
